calculator/src/Components/Keypad.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+class Keypad extends React.Component {
```

Comment:
Because this component doesn't track state, think about updating to be a functional component